test/ussdapp.test.js

```diff
@@ -22,26 +22,18 @@ describe("UoP TB registration/quiz app", function() {
                     randomize_quizzes: false,
                     randomize_questions: false,
                     services: {
-                        identities: {
+                        "identities": {
                             api_token: 'test_token_identities',
                             url: "http://localhost:8001/api/v1/"
                         },
-                        registrations: {
+                        "registrations": {
```

Comment:
last little one. Make this `hub` and the places that it affects (from previous issues).